docs/PR_Guidelines.md

```diff
@@ -0,0 +1,13 @@
+# Guidelines for creating a good pull request
```

Comment:
Should we expand a section on how to test locally to save cycles on CI? For example, PRs should be submitted once code style is verified to prevent spending hours in CI just to realize some formatting is not good. Running local tests locally should save CI time too, but we briefly mention how to do it

Reply:
I can add a doc. 
If you have a Linux box, that should be very easy to run the Linux build locally. Because we use docker.

Reply:
I agree it is easy, but should be documented so beginners can have a good start point, especially when geetting started on Linux and/or ORT

Reply:
Added task and assigned to @snnn .

Reply:
1. Download the azcopy v10 tool from https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10
2. Use azcopy tool to download the latest test data from https://onnxruntimetestdata.blob.core.windows.net/models/20191107.zip,then unzip it to /data/models
3. mkdir build
4. tools/ci_build/github/linux/run_dockerbuild.sh -o ubuntu16.04 -d cpu -r `pwd`/build -x "--use_mklml --use_llvm --use_nuphar --use_dnnl --use_tvm --build_wheel --build_java --enable_language_interop_ops --use_featurizers --wheel_name_suffix featurizers"

Reply:
We can add these instructions somewhere on the BUILD.md page. Thx.